docs/docs/pyproject.md

```diff
@@ -243,15 +243,32 @@ mysqlclient = { version = "^1.3", optional = true }
 [tool.poetry.extras]
 mysql = ["mysqlclient"]
 pgsql = ["psycopg2"]
+databases = ["mysqlclient", "psycopg2"]
 ```
 
-When installing packages, you can specify extras by using the `-E|--extras` option:
+When installing packages with poetry, you can specify extras by using the `-E|--extras` option:
 
 ```bash
 poetry install --extras "mysql pgsql"
 poetry install -E mysql -E pgsql
 ```
 
+When installing or specifying poetry-built packages, the extras defined in this section can be activated 
+as described in [PEP 508](https://www.python.org/dev/peps/pep-0508/#extras).
+
+For example, the dependencies required by the `databases` extra can be activated when installing the 
```

Comment:
Is "activated" the right word here? Doesn't "installed" more accurately describe what actually happens?

Reply:
Good catch! Will get it fixed